x-pack/test/apm_api_integration/basic/tests/service_overview/dependencies.ts

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
I think this test should be placed inside `/Services` not `/service_overview`, since we don't have service_overview API.